writer/writer/redis_backend/redis_messaging_backend_service.py

```diff
@@ -19,6 +19,11 @@ class RedisMessagingBackendService(Messaging):
 
     def handle_events(self, events: List[BaseDbEvent], position: int) -> None:
         modified_fqfields = self.get_modified_fqfields(events, position)
+        # only log, if it is not the startup data.
+        if "organisation/1/id" not in modified_fqfields:
```

Comment:
Mmh, I don't like making specific exceptions like this... Is this just for avoiding the huge bulk of logs at the startup? Maybe add a flag a lá `verbose=False` to the `write` method, which is called in `create_initial_data.py`?

Reply:
I was fine with it since it is a `debug` log. We can also remove it, if we get a real set of start-data, which will not contain as much data as the current example data.

Reply:
Pushed my suggestion. I think it's just cleaner ;)